doc/source/how_to.rst

```diff
@@ -0,0 +1,213 @@
+How to
+======
+                            
+
+How to reset your password
+--------------------------
+
+**NB**: This section is only available if the use of emails to login is
+activated in the global configuration. It is not the case by default.
+
+If you forgot your password or if you need to set one, first call the
+forgotten_password endpoint:
+
+.. raw:: html
+
+   <!-- It is python, just doing this so that test-integration does not run this code (need mail config to run)  -->
```

Comment:
If it's rst, it's not tested anymore. so javascript -> python and remove the comment